src/generated/api.d.ts

```diff
@@ -0,0 +1,152 @@
+import http = require("http");
```

Comment:
since this is auto generated, can this not be in src control and get generated as part of build process?

Reply:
> since this is auto generated, can this not be in src control and get generated as part of build process?

It was done but during the development we ended up moving it out (https://github.com/octokit/webhooks.js/pull/113/commits/f9480b417b649de3f433b8177e0bc24b9467dbac) of `src/` folder.

Context here: https://github.com/octokit/webhooks.js/pull/127#issuecomment-620301310

Reply:
> can this not be in src control

It's not a comparable to a built artefact like JS files in project written in TypeScript. This file is part of the actual source, it is automated based on https://github.com/octokit/webhooks via automated pull requests. In these pull requests we want to be able to review the differences in the source files that we publish to npm